Sources/RoutingKit/AnyRouter.swift

```diff
@@ -1,4 +1,5 @@
 /// A router that performs type erasure by wrapping another router.
+@available(*, deprecated, message: "This type is no longer useful in Swift 5.7")
```

Comment:
Perhaps we can mention the alternative too? it's `any Router<Output>` as far as i can tell.

Reply:
Not exactly the case, and the point is there's no need for an alternative at all, nor is it necessarily clear that there's exactly one alternate choice, especially since this hasn't been used for a long time.